mlrun/common/model_monitoring/stores/sql_model_endpoint_store.py

```diff
@@ -20,11 +20,8 @@
 import pandas as pd
 import sqlalchemy as db
 
-import mlrun
 import mlrun.common.model_monitoring as model_monitoring_constants
-import mlrun.model_monitoring.model_endpoint
 import mlrun.utils.model_monitoring
-import mlrun.utils.v3io_clients
 from mlrun.api.db.sqldb.session import create_session, get_engine
```

Comment:
We are reusing these functions for the SQL store abstraction (#2685). Instead of redefining `mlrun/api/db/sqldb/session.py` for the monitoring SQL store, we have decided to reuse these functions but that means that we need to import the code from the API. 
Therefore, we have 2 options here:
1 - duplicate the code within `mlrun/api/db/sqldb/session.py` for the monitoring SQL store (create a new file under this path - `mlrun/common/stores/sql_session.py`). 
2 - move the current `mlrun/api/db/sqldb/session.py` file to common. Note that it doesn't import any other packages from the API side and is being used only by the API and the monitoring SQL store abstraction. 

Let me know what do you think is the best approach here. I'm in favor of option (2) but we need to consider that it will affect other places in the API side while option (1) only affects the SQL store abstraction.